OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -34,6 +34,19 @@ public class UpdatePath
 			Justification = "Extracting update lists to temporary variables obfuscates the definitions.")]
 		static readonly UpdatePath[] Paths =
 		{
+			new UpdatePath("release-20171014", "release-20180218", new UpdateRule[]
```

Comment:
Can we please comment out this block or maybe rename it to `incomplete-release-20171014` to make it clear that this is not ready yet?